src/common/quantile.cu

```diff
@@ -668,7 +712,7 @@ void SketchContainer::MakeCuts(HistogramCuts* p_cuts) {
 
     if (IsCat(d_ft, column_id)) {
       assert(out_column.size() == in_column.size());
-      out_column[idx] = in_column[idx].value;
+      out_column[idx] = idx;
```

Comment:
What does this line do? Isn't this a no-op?

Reply:
Assign idx to output, it's not a no op.

Reply:
So `out_column` is going to be `[0, 1, 2, 3, ...]`, like `iota`?

Reply:
Yes. [0, max_cat] inclusive